Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -567,19 +567,35 @@ class ToggleOnboarding: HiddenSetting {
 
 class LeanplumStatus: HiddenSetting {
     let lplumSetupType = LeanPlumClient.shared.lpSetupType()
-
     override var title: NSAttributedString? {
-        return NSAttributedString(string: "Leamplum Status: \(lplumSetupType) | Started: \(LeanPlumClient.shared.isRunning())", attributes: [NSAttributedString.Key.foregroundColor: UIColor.theme.tableView.rowText])
+        return NSAttributedString(string: "Leamplum Status: \(lplumSetupType) | Started: \(LeanPlumClient.shared.isRunning())\nLeanplum Devide ID - \(LeanPlumClient.shared.leanplumDeviceId ?? "")", attributes: [NSAttributedString.Key.foregroundColor: UIColor.theme.tableView.rowText])
```

Comment:
Device is spelled wrong; first item should be `LP Setup Type:`, or just `LP Setup:` because the value makes the key clear, i.e. this reads clearly enough `LP Setup: production`
use a colon each time between key/value pairs (not a dash);
this seems too long, try "LP Setup: xxx | Started: xxx | Device ID: xxx"